src/go/rpk/pkg/cli/cmd/status.go

```diff
@@ -96,19 +96,30 @@ func executeStatus(
 	t.Append(getVersion())
 
 	providerInfoRowsCh := make(chan [][]string)
-	metricsRowsCh := make(chan [][]string)
+	metricsCh := make(chan metricsResult)
 	confRowsCh := make(chan [][]string)
 	kafkaRowsCh := make(chan [][]string)
 
+	go getMetrics(fs, mgr, timeout, *conf, send, metricsCh)
+
+	metricsRes := <-metricsCh
+
+	if send && metricsRes.metrics != nil {
+		err := sendMetrics(*conf, metricsRes.metrics)
+		if err != nil {
+			return fmt.Errorf("Error sending metrics: %v", err)
+		}
+		return nil
```

Comment:
Is not returning `err` Intentional?

Reply:
I wanted to wrap the error in a more human-friendly message, hence I'm not returning it directly.

Reply:
This is fine. I'm not sure why I questioned it.